examples/tutorials/frames.py

```diff
@@ -105,6 +105,6 @@
     projection="X10c/8c",
     # Plot axis, tick marks, and axis labels on the west/left and south/bottom axes
     # Plot axis and tick marks on the north/top and east/right axes
-    frame=["WSne", "x+lx-axis", "y+ly-axis"],
+    frame=["af", "WSne", "x+lx-axis", "y+ly-axis"],
```

Comment:
This is what I prefer:
```suggestion
    frame=["WSne", "xaf+lx-axis", "yaf+ly-axis"],
```